### PR TITLE
[FIX] account_credit_control: Missing read access

### PR DESCRIPTION
**Pull request by rven**
Missing read access on model credit.control.policy for group_account_user

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Why do you need that permission?

**Comment by rven** [comment]
@pedrobaeza 
In "res_config_settings_view.xml" some settings are added to the configuration screen, with groups="account.group_account_user". These settings contain a field credit_policy_id, which is a many2one field to model "credit.control.policy". 

However, the security lines for this model are these:

account_credit_control.ir_model_access_284,credit_control_manager_policy,account_credit_control.model_credit_control_policy,group_account_credit_control_manager,1,1,1,1
account_credit_control.ir_model_access_285,credit_control_user_policy,account_credit_control.model_credit_control_policy,group_account_credit_control_user,1,0,0,0
account_credit_control.ir_model_access_286,credit_control_info_policy,account_credit_control.model_credit_control_policy,group_account_credit_control_info,1,0,0,0
So only groups "group_account_credit_control_manager", "group_account_credit_control_user" and "group_account_credit_control_info" have access to this model. 

When a user belongs to group "account.group_account_user", but does NOT belong to one of those three groups, an error is raised when this user visits the configuration settings screen (because the field credit_policy_id is shown, but the user has no access to this model!).

**Comment by pedrobaeza** [comment]
OK, I see. You can access configuration without being accounting adviser. Isn't better to protect that field with an allowed group?

**Comment by rven** [comment]
I don't know what the intention was of the person who created this module. But from my stand of view it should be logic that a user with the rights **Show Full Accounting Features** _(account.group_account_user)_, also should be able to _see_ this field.

**Comment by pedrobaeza** [comment]
I would say no, as there are specific permissions for users and managers of credit control independently from accounting permissions.

**Comment by rven** [comment]
@pedrobaeza I changed the security on the view itself to the credit control manager.
This way the fields are not blocking the user if he just has the rights **Show Full Accounting Features** (account.group_account_user).

**Comment by pedrobaeza** [comment]
Why making some unrelated changes like removing `string` attribute here in this PR?

**Comment by rven** [comment]
I always run the pylint and pep8 check when doing changes, so I always optimize the quality of the code.
Isn't that a good idea?

**Comment by pedrobaeza** [comment]
No on a simple fix. And more that most of these changes are opinionated ones. There's no problem in the previous form. I would say even more: I prefer the other style.

File: account_credit_control/models/credit_control_run.py [hunk 1]
```diff
@@ -113,8 +113,7 @@ def _generate_credit_lines(self):
             else:
                 report += _(
                     "Policy \"<b>%s</b>\" has not generated any "
-                    "Credit Control Lines.<br/>" % policy.name
-                )
+                    "Credit Control Lines.<br/>") % policy.name
```
Comment (pedrobaeza) [inline account_credit_control/models/credit_control_run.py:116]: For example here: there's no problem with this previous form and I prefer it.

**Pull request closed without merging**